src/main/java/at/hannibal2/skyhanni/features/inventory/ItemPickupLog.kt

```diff
@@ -122,7 +124,11 @@ object ItemPickupLog {
         if (!isEnabled() || !config.sack) return
 
         event.sackChanges.forEach {
-            val itemStack = (it.internalName.getItemStack())
+
+            // TODO this should not need to be done here but the whole internal name resolving needs a rework and this fixes it for now
+            val internalName = if (it.internalName == HAY_BALE) HAY_BLOCK else it.internalName
```

Comment:
code duplication of https://github.com/hannibal002/SkyHanni/blob/7c2d4ed566246513bc438272e436627412147d62/src/main/java/at/hannibal2/skyhanni/utils/ItemNameResolver.kt#L66